cmd/acr/purge.go

```diff
@@ -659,11 +715,14 @@ func buildRegexFilter(expression string, regexpMatchTimeoutSeconds uint64) (*reg
 	return regexp, nil
 }
 
-// In order to parse the content of a mutliarch manifest string the following structs were defined.
-type multiArchManifest struct {
-	Manifests     []manifest `json:"manifests"`
-	MediaType     string     `json:"mediaType"`
-	SchemaVersion int        `json:"schemaVersion"`
+// customManifest defines a customized struct for manifests
+// which is used to parse the content of a multiarch manifest,
+// or references a subject
+type customManifest struct {
+	Manifests     []manifest     `json:"manifests"`
+	MediaType     string         `json:"mediaType"`
+	SchemaVersion int            `json:"schemaVersion"`
```

Comment:
Looks like this is not in use. Should we remove it?

Reply:
removed `MediaType` and `SchemaVersion`